test/OpenTelemetry.Tests/Trace/LinkTest.cs

```diff
@@ -83,7 +83,7 @@ public void Equality()
             Assert.True(link1.Equals(link3));
         }
 
-        [Fact]
+        [Fact(Skip = "ActivityLink.Equals is broken in DS7 preview: https://github.com/dotnet/runtime/issues/74026")]
```

Comment:
🙈 🙉 🙊

Reply:
FYI https://github.com/dotnet/dotnet-api-docs/pull/8316

We discussed it internally and essentially no one is sure what it means for two ActivityLinks to be equal so we're going to leave it broken/undefined.